app/models/etsource/loader.rb

```diff
@@ -46,8 +46,11 @@ def presets
 
     def area_attributes(area_code)
       @area_attributes[area_code] ||= begin
-        area_attr = Atlas::Dataset.find(area_code).to_hash
+        area = Atlas::Dataset.find(area_code)
+        area_attr = area.to_hash
+        area_attr[:is_derived] = true if area.is_a?(Atlas::Dataset::DerivedDataset)
```

Comment:
> I only want to have this key in the hash if it is true, but not if it is false

That's because of [this line](https://github.com/quintel/etmodel/pull/2313/files#diff-f8737ffcf0d1a40db31e52e65222b993R47) I guess? But I think it's also perfectly acceptable to have `false` there for `is_derived?` I guess. Unless it's used in other places?